src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -361,16 +370,18 @@ protected function load($paths)
             array_unique(array_merge($this->loadedPaths, $paths))
         );
 
-        $namespace = $this->app->getNamespace();
-
         foreach (Finder::create()->in($paths)->files() as $file) {
-            $command = $this->commandClassFromFile($file, $namespace);
-
-            if (is_subclass_of($command, Command::class) &&
-                ! (new ReflectionClass($command))->isAbstract()) {
-                Artisan::starting(function ($artisan) use ($command) {
-                    $artisan->resolve($command);
-                });
+            $command = $this->commandClassFromFile($file);
+
+            try {
+                if (is_subclass_of($command, Command::class) &&
+                    ! (new ReflectionClass($command))->isAbstract()) {
+                    Artisan::starting(function ($artisan) use ($command) {
+                        $artisan->resolve($command);
+                    });
+                }
+            } catch (ErrorException|ReflectionException) {
```

Comment:
This was added to avoid diffing composer-excluded files and directories, not a big fan of it though